examples/static_react_task/webapp/package.json

```diff
@@ -26,7 +26,7 @@
     "file-loader": "^6.0.0",
     "style-loader": "^0.23.0",
     "url-loader": "^2.0.1",
-    "webpack": "^4.19.1",
+    "webpack": "^5.0.0-rc.6",
```

Comment:
I always thought this would be a big upgrade, glad it wasn't too bad

Reply:
Yeah the disruption is pretty minor across the board, I'm happy to help if downstream projects hit an issue after this lands.